src/lib/lightnet.test.js

```diff
@@ -0,0 +1,3 @@
+describe('lightnet.js', () => {
+  it.todo('should be correct');
```

Comment:
I think we should probably not include this if we don't intend to write it out

Reply:
Tests will be added later as part of the work on https://github.com/o1-labs/zkapp-cli/issues/509.